### PR TITLE
chore(content-uploader): Add visual regression tests

### PR DESCRIPTION
**Pull request by greg-in-a-box**
image

error single upload removed for being too unstabled

### CODE REVIEW

File: src/elements/content-uploader/DroppableContent.js [hunk 1]
```diff
@@ -52,26 +52,33 @@ const dropDefinition = {
...
-    ({ canDrop, isOver, isTouch, view, items, addFiles, onClick, isFolderUploadEnabled }: Props) => {
-        const handleSelectFiles = ({ target: { files } }: any) => addFiles(files);
-        const hasItems = items.length > 0;
+const DroppableContent = makeDroppable(dropDefinition)(({
+    canDrop,
+    isOver,
+    isTouch,
+    view,
+    items,
+    addFiles,
```
Comment (jpan-box) [inline src/elements/content-uploader/DroppableContent.js:61]: abc? cc trevor

File: src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        token: global.TOKEN,
+    },
+    parameters: {
+        ...defaultVisualConfig.parameters,
+        msw: {
+            handlers: [
+                http.options(`${DEFAULT_HOSTNAME_API}/2.0/files/content`, () => {
+                    return HttpResponse.json({});
+                }),
+                http.post(`${DEFAULT_HOSTNAME_UPLOAD}/api/2.0/files/content`, () => {
```
Comment (jpan-box) [inline src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js:214]: i know that having the /api/ in the path vs subdomain works, but should it? or is there any reason to have the /api/ in there?

Comment (greg-in-a-box) [reply src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js:214]: There is only one location outside of this file that has the work api next to the word, dont know if its worth actually creating a new constant for it

Comment (jpan-box) [reply src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js:214]: i think it's more "proper" from a backend routing perspective to _not_ have the /api/ in the path. would you be ok to remove it, if the tests still pass?

Comment (greg-in-a-box) [reply src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js:214]: /api/ is part of the official route its mocking

Comment (jpan-box) [reply src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js:178]: yup confirmed

**Review by jpan-box — APPROVED** [review]

**Review by tjiang-box — DISMISSED** [review]

File: src/elements/content-uploader/UploadInput.js [hunk 3]
```diff
@@ -36,6 +36,7 @@ const UploadInput = ({
         <label className={inputLabelClass} onKeyDown={onKeyDown} tabIndex={0}>
             {inputLabel}
```
Comment (jstoffan) [inline src/elements/content-uploader/UploadInput.js:37]: Can we select this input using user-facing label text rather than a `data-testid` selector?

Comment (greg-in-a-box) [reply src/elements/content-uploader/UploadInput.js:41]: going to add to an aria-label and getByRole instead

File: src/elements/content-uploader/UploadInput.js [hunk 4]
```diff
@@ -36,10 +36,11 @@ const UploadInput = ({
...
             <input
-                ref={inputRef}
+                data-testid="upload-input"
                 directory={isFolderUpload ? '' : undefined}
-                multiple={isMultiple}
                 onChange={handleChange}
+                multiple={isMultiple}
```
Comment (jstoffan) [inline src/elements/content-uploader/UploadInput.js:42]: Misordered?

**Review by jstoffan — APPROVED** [review]

**Review by jpan-box — DISMISSED** [review]

File: src/elements/content-uploader/UploadInput.js (hunk 4 above)
Comment (greg-in-a-box) [reply src/elements/content-uploader/UploadInput.js:41]: fixed

**Review by jstoffan — APPROVED** [review]

**Review by tjiang-box — APPROVED** [review]

**Review by jpan-box — APPROVED** [review]

**Pull request merged**